sdk/spring/azure-spring-cloud-autoconfigure/README.md

```diff
@@ -91,7 +91,7 @@ Please follow [instructions here][contributing_md] to build from source or contr
 [event_hubs]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/azure-spring-cloud-starter-eventhubs
 [event_hubs_kafka]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/azure-spring-cloud-starter-eventhubs-kafka
 [service_bus]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/azure-spring-cloud-starter-servicebus
-[storage]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/azure-spring-starter-storage
+[storage]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/azure-spring-boot-starter-storage
```

Comment:
Not sure if Azure Spring Boot Storage starter is suitable to be here, I think we could move this entry to azure-spring-boot's README?

Reply:
Done.